projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/tabbing-product-configuration.e2e-spec.ts

```diff
@@ -0,0 +1,81 @@
+import * as configuration from '../../helpers/product-configuration';
+import * as configurationOverview from '../../helpers/product-configuration-overview';
+import { tabbingOrderConfig as tabConfig } from '../../helpers/accessibility/tabbing-order.config';
+import { verifyTabbingOrder } from '../../helpers/accessibility/tabbing-order';
+
+const testProduct = 'CONF_CAMERA_SL';
+
+const configurator = 'CPQCONFIGURATOR';
+
+const containerSelectorConfigForm = '.VariantConfigurationTemplate';
+const containerSelectorOverviewForm = '[class*=VariantConfig]';
+
+function goToConfigPage(configuratorType, product) {
+  cy.visit(
+    `/electronics-spa/en/USD/configure${configuratorType}/product/entityKey/${product}`
+  ).then(() => {
+    configuration.isConfigPageDisplayed();
+  });
+}
+
+context('Product Configuration', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  describe('Product Config Tabbing', () => {
+    it('should allow to navigate with tab key', () => {
+      goToConfigPage(configurator, testProduct);
+      configuration.isGroupMenuDisplayed();
+      configuration.isConfigHeaderDisplayed();
+
+      verifyTabbingOrder(
+        containerSelectorConfigForm,
+        tabConfig.productConfigurationPage
+      );
+
+      configuration.selectAttribute('CAMERA_MODE', 'radioGroup', 'P');
```

Comment:
Define constants to enable reuse of the values: '**CAMERA_MODE**', '**radioGroup**', '**P**'

Reply:
done.